source/css/blog/_layout.scss

```diff
@@ -1,131 +0,0 @@
-.blog {
-  iframe {
-    margin-bottom: 4.875rem;
-  }
-  
-  @include respond-to(md, min) {
-    padding-top: $gutter;
-  }
-
-  &__header {
-    margin-bottom: $gutter;
-
-    @include respond-to(md, min) {
-      margin-bottom: $gutter*1.5;
-    }
-
-    @include respond-to(lg, min) {
-      margin-bottom: $gutter*2;
-    }
-  }
-
-  .alternate {
-    @include respond-to(sm, max) {
-      background-color: white; // blog gets a white background
-    }
-  }
-}
-// page specific
-.intro {
-  &.centered {
-    text-align: center;
-  }
-
-  .heading {
-    h1 {
-      margin-top: 0;
-    }
-  }
-
-  p {
-    max-width: 40rem;
-    margin: 0 auto 1.625rem;
-
-    &:last-child {
-      margin-bottom: 0;
-    }
-  }
-}
-
-// Images
-img {
-  max-width: 100%;
-
-  &.small {
-    max-width: 220px;
-  }
-}
```

Comment:
FYI @maisiembleton this was that image thing, this is global.  There's probably a few other rules in here that were global…accidentally or otherwise.

Reply:
ahh sweet, I re-added those back in, minus the` .blog { ... } `part. I actually can see that `intro` is used in `page-xero.html.erb` too

Reply:
though I do wonder if it needs to be nested in a blog folder? does that impact the structure of the css? It's been a while since I've delved into scss/css outside of styled components